pubsub/README.md

```diff
@@ -92,14 +90,23 @@ the peer won't be able to send identical messages (e.g. keepalives) within the
 timecache interval, as they will get rejected as duplicates."
 
 The `topicIDs` field specifies a set of topics that this message is being
-published to. To prevent the size of this field from growing indefinitely,
-it is suggested that that the application using an implementation of this
-library spec should have some kind of storage economics (see e.g.
-[here](https://ethresear.ch/t/draft-position-paper-on-resource-pricing/2838).
+published to.
 
 Note that messages are currently *not* signed. This will come in the near
 future.
 
+The size of the `Message` should be limited, say to 1 MiB, but could also
+be configurable, for more information see 
+[issue 118](https://github.com/libp2p/specs/issues/118), while messages should be
+rejected if they are over this size (this isn't done as of Dec 2018).
```

Comment:
this is incorrect, messages over 1MB _are_ rejected, by virtue of a read error and closing the stream.

Reply:
at least in go-libp2p-pubsub.